control-plane/csi-driver/src/bin/node/nodeplugin_grpc.rs

```diff
@@ -54,6 +65,73 @@ impl NodePlugin for NodePluginSvc {
             device_path: device.devname(),
         }))
     }
+
+    async fn cleanup(
+        &self,
+        request: Request<CleanupRequest>,
+    ) -> Result<Response<CleanupReply>, Status> {
+        if !self.stale_entry_cleanup {
+            return Ok(Response::new(CleanupReply {}));
+        }
+        let volume_id = request.into_inner().volume_id;
+        debug!("Starting cleanup for volume: {volume_id}");
+
+        let nqn = format!("{}:{volume_id}", nvme_target_nqn_prefix());
+        match Subsystem::try_from_nqn(&nqn) {
+            Ok(subsystem_paths) => {
+                for subsystem_path in subsystem_paths {
+                    debug!(
+                        "Processing subsystem path: addr: {:?}, transport: {}, state: {}",
+                        subsystem_path.address, subsystem_path.transport, subsystem_path.state
```

Comment:
```suggestion
                        "Processing subsystem: {subsystem_path:?}, 
```

Reply:
Isn't the addr and path important here ? nqn is kind of same for everyone

Reply:
> Isn't the addr and path important here ? nqn is kind of same for everyone

It is. I thought the Debug trait might take care of that, but may be it won't as it's only derived?
You can retain the print as-is then.

Reply:
hmm, Debug is derived for `SubsystemAddr` too so might just work the way suggested earlier. You can try and take a call as you deem fit.

Reply:
Done, as you suggested.